python/lsst/meas/deblender/SConscript

```diff
@@ -1,3 +1,3 @@
 # -*- python -*-
 from lsst.sconsUtils import scripts
-scripts.BasicSConscript.pybind11(['baseline'])
+scripts.BasicSConscript.pybind11(['baselineUtils'])
```

Comment:
Please use `addUnderscore=False` in this call; that will let you call the module just `baselineUtils` everywhere else (and leave us only one place to change after not-adding-underscores becomes the default).